src/baldr/tilehierarchy.cc

```diff
@@ -1,44 +1,104 @@
 #include "baldr/tilehierarchy.h"
 #include "baldr/graphtileheader.h"
+#include "config.h"
 #include "midgard/vector2.h"
 
 using namespace valhalla::midgard;
 
 namespace valhalla {
 namespace baldr {
 
-const std::vector<TileLevel>& TileHierarchy::levels() {
-  // Static tile levels
-  static const std::vector<TileLevel> levels_ = {
+static std::vector<TileLevel> levels_;
+static std::vector<TileLevel> transit_levels_;
+
+void getLevelsfromConfig() {
```

Comment:
what I'd like to change is the coding style and would ask to make it more valhalla like:
- either camel case with the first letter capitalized or snake case
- no `bSizesInConfigOk`, `tilerowsvector` etc indicating the type as part of the variable name
- variable names always snake case (we'd like that for functions too actually, but somehow java-ish google style crept in at some point)